public/app/features/dashboard/state/PanelModel.ts

```diff
@@ -390,7 +415,22 @@ export class PanelModel {
 
   setTransformations(transformations: DataTransformerConfig[]) {
     this.transformations = transformations;
-    this.getQueryRunner().setTransformations(transformations);
+  }
+
+  replaceVariables(value: string, extraVars?: ScopedVars, format?: string) {
+    let vars = this.scopedVars;
+    if (extraVars) {
+      vars = vars ? { ...vars, ...extraVars } : extraVars;
+    }
+    return templateSrv.replace(value, vars, format);
+  }
+
+  updateQueryRunnerFieldOverrides() {
+    if (!this.plugin) {
+      return;
+    }
+
+    this.getQueryRunner().resendLastResult();
```

Comment:
Nost sure this makes that much sense as a function as it does not do what it says, should maybe be re-named resendLastResult